src/locale/properties/layout/layout.en.properties

```diff
@@ -6,10 +6,11 @@ common.old_browser_3=supported browser
 footer.copyright_cc0_1=The text of this website is published under a
 footer.copyright_cc0_3=Images and marks are subject to copyright and trademark protection
 common.period=.
-footer.aboutOrcid=About ORCID
 footer.privacypolicy=Privacy Policy
 footer.termsofuse=Terms of Use
 footer.contactus=Contact us
+footer.accessibility=Accessibility Statement
```

Comment:
The new properties are missing in the testing languages

Reply:
CI is supposed to generate those let's wait to see if it does